hooks/useMaskParam.ts

```diff
@@ -0,0 +1,21 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+import { useMemo } from 'react';
+
+export function useMaskParam(): boolean {
```

Comment:
I guess if we call this `useIsMaskParamAvailable` then it makes more sense based on the hook's output.

Reply:
I'm not sure the suggested name is better 🤔 Since the hook returns the parsed `mask` param, I think the name is fine? 🤔

Reply:
Ah. I see! My bad :) so the value is either `true` or `false`. Got it :)